firmware/src/main.rs

```diff
@@ -27,22 +27,13 @@ fn main() -> ! {
 
 	static EXECUTOR: StaticCell<Executor> = StaticCell::new();
 	EXECUTOR.init(Executor::new()).run(move |spawner| {
-		spawner.spawn(network_task()).unwrap();
+		spawner
+			.spawn(self::networking::wifi::ඞ::network_task())
```

Comment:
I prefer to have the spawner call the async tasks in `main.rs`, and then those can delegate out to the actual cfg-gated tasks. See `imu_task` for example.